oauthenticator/globus.py

```diff
@@ -307,12 +303,8 @@ def user_info_to_username(self, user_info):
         if self.identity_provider and domain != self.identity_provider:
             raise HTTPError(
                 403,
-                'This site is restricted to {} accounts. Please link your {}'
-                ' account at {}.'.format(
-                    self.identity_provider,
-                    self.identity_provider,
-                    'globus.org/app/account',
-                ),
+                f"This site is restricted to {self.identity_provider} accounts. "
+                "Please link your account at globus.org/app/account.",
```

Comment:
Since the content is being changed here, it's worth noting [this leads to an error page:](https://globus.org/app/account)

> The legacy Console is no longer supported, please update your bookmarks.
> [Back to the Globus Web Application](https://app.globus.org/console/dashboard)

Is that a suitable replacement destination?

Reply:
I googled on app.globus.org account page, and it seems [from here](https://docs.globus.org/how-to/link-to-existing/) the replacement should be app.globus.org/account instead of globus.org/app/account

Reply:
```suggestion
                "Please link your account at app.globus.org/account.",
```